src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig

```diff
@@ -48,14 +48,13 @@
         {% set checked = (form.vars.submitted_values is defined and submitted_values[child.id_category] is defined) ? 'checked="checked"' : '' %}
 
          <div class="radio">
-             <label class="category-label" for="form[{{ form.vars.id }}][tree]">{{ child.name }}
-                 <input
-                   type="radio"
-                   name="form[{{ form.vars.id }}][tree]"
-                   value="{{ child.id_category }}" {{ checked }}
-                   class="category pull-right"
-                 />
-             </label>
+             <label class="category-label" for="form[{{ form.vars.id }}][tree]"> {{ child.name }} </label>
+             <input
+                 type="radio"
+                 name="form[{{ form.vars.id }}][tree]"
+                 value="{{ child.id_category }}" {{ checked }}
+                 class="category pull-right"
+             />
```

Comment:
I agree, I've read it in the prestashop coding standards document. 
FYI, I've just moved the input outside of the label.
Anyway, I will remove the self-closing tag.

Thanks !